### DISCUSSION TITLE
Environment variables for MJPEG stream

### DISCUSSION THREAD

**Discussion by joao-borges-miw** [danielpaulus/go-ios#522] [Q&A]
Hi,

Can someone point me to where I can find all the env vars possible for the MJPEG stream?
I googled it and couldn't find anything.

Thank you

**Comment by shamanec** [comment]
Which MJPEG stream? the `go-ios` one does not use env vars
`ios screenshot [options] [--output=<outfile>] [--stream] [--port=<port>]  Takes a screenshot and writes it to the current dir or to <outfile>  If --stream is supplied it starts an mjpeg server at 0.0.0.0:3333. Use --port to set another port.`
